app/models.py

```diff
@@ -15,3 +17,41 @@ class User(AbstractBaseUser):
 
     def __str__(self):
         return self.contact
+
+
+class FriendRequest(models.Model):
+    from_user = models.ForeignKey(
+        User,
+        on_delete=models.CASCADE,
+        related_name="user_sent",
+    )
+    to_user = models.ForeignKey(
+        User,
+        on_delete=models.CASCADE,
+        related_name="user_received",
+    )
+
+    message = models.TextField(blank=True)
```

Comment:
Could you explain the use of this field?

Reply:
I thought we could use to send custom friend request messages else a default request message.

Reply:
Remove it for now, we can add it later if needed